sunpy/timeseries/timeseriesbase.py

```diff
@@ -281,10 +281,11 @@ def _repr_html_(self):
         # Create the timeseries plots for each channel as a panel in one
         # figure. The color list below is passed to both timeseries and
         # histogram plotting methods for consistency.
-        cols = ['b', 'g', 'r', 'c', 'm', 'y', 'tab:blue', 'tab:orange',
-                'tab:red', 'tab:purple', 'tab:brown', 'tab:pink', 'tab:gray',
-                'tab:green', 'tab:olive', 'tab:cyan', 'palegreen', 'pink'
-                ]
+        cols = np.concatenate([
+            plt.cm.tab20(np.linspace(0, 1, 20)),
+            plt.cm.tab20b(np.linspace(0, 1, 20)),
+            plt.cm.tab20c(np.linspace(0, 1, 20))
```

Comment:
What would a time-series with 20 columns look like now when plotted?

This I assume puts again a max limit at 60 columns?

Reply:
In the codebase `plot, peek` uses color in circular way(repeats after 10th column)
I think we should do the same for `__repr_html__`  .
and  should use the color(already predefined)  list or different 60 colors(using colormaps) in circular way.
What should works best ??

```
import numpy as np
import pandas as pd
from sunpy.timeseries import TimeSeries
from astropy.time import Time
import astropy.units as u

times = pd.date_range("now", periods=50, freq="T")
data = np.random.rand(50, 20)
columns = [f'col_{i+1}' for i in range(20)]  

df = pd.DataFrame(data, index=times, columns=columns)
header = {}
units = {col:u.min for col in columns}
ts = TimeSeries(df, header, units)

ts.peek(figsize = (20,10))
```

![image](https://github.com/user-attachments/assets/7c0d137d-d2e0-4fce-a198-15c9e8d42787)

Reply:
IF the other methods are just cycling, we should just do the same for now.

Reply:
I have made the changes 
```
>>> a = [1,2,3]
... for i in range(12):
...     print(a[i%3], end=' ')
...     
1 2 3 1 2 3 1 2 3 1 2 3 >>> 
```
This way cols will be used in circular way